app/controllers/v0/form1010_ezrs_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module V0
+  class Form1010EzrsController < ApplicationController
+    skip_before_action :authenticate, only: %i[create]
+    before_action :load_user, only: %i[create]
```

Comment:
get rid of these 2 lines, they are only necessary for the 1010ez because it supports unauth and auth users

Reply:
Removed!